### PR TITLE
Add Azure Deployment documentation

### PR DESCRIPTION
**Pull request by guy-har**

### CODE REVIEW

File: docs/Gemfile.lock [hunk 1]
```diff
@@ -215,6 +215,8 @@ GEM
     nokogiri (1.11.0)
       mini_portile2 (~> 2.5.0)
       racc (~> 1.4)
+    nokogiri (1.11.0-arm64-darwin)
```
Comment (ozkatz) [inline docs/Gemfile.lock:218]: remove this please

File: docs/branching/index.md [hunk 2]
```diff
@@ -2,6 +2,6 @@
 layout: default
 title: Branching Model
 description: At its core, lakeFS uses a Git-like branching model that scales to Petabytes of data by utilizing S3 or GCS for storage.
-nav_order: 3
+nav_order: 25
```
Comment (ozkatz) [inline docs/branching/index.md:5]: why?

File: docs/community.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 layout: default
 title: Community
 description: lakeFS community. Join the community on the lakeFS slack channel and feel free to ask questions and get help.
-nav_order: 10
+nav_order: 50
```
Comment (ozkatz) [inline docs/community.md:5]: why?

File: docs/contributing.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 layout: default
 title: Contributing
 description: lakeFS community welcomes your contribution. To make the process as seamless as possible, we recommend you read this contribution guide.
-nav_order: 10
+nav_order: 55
```
Comment (ozkatz) [inline docs/contributing.md:5]: whyyyy

**Review by ozkatz — CHANGES_REQUESTED** [review]
Please remove the updated dependencies

File: docs/branching/index.md (hunk 2 above)
Comment (guy-har) [reply docs/branching/index.md:5]: In order to insert the `Azure deployment` page right after `AWS deployment'  I needed to shift all the other pages by one, So I multiplied them by five making it more convenient for the following cases.

**Review by ozkatz — APPROVED** [review]
Looks great! thanks

**Pull request merged**